user/qbe/patches/fix_asm_keyword.patch

```diff
@@ -0,0 +1,62 @@
+--- a/arm64/emit.c	2024-01-29 10:24:41.000000000 +0100
```

Comment:
instead of this patch you could also set `tool_flags = {"CFLAGS": ["-std=c99"]}` in the template (which is what the upstream Makefile sets anyway) and remove this patch entirely, but this patch is also fine